packages/react/context-menu/src/ContextMenu.tsx

```diff
@@ -35,7 +35,7 @@ interface ContextMenuProps {
 const ContextMenu: React.FC<ContextMenuProps> = (props) => {
   const { children, onOpenChange, dir, modal = true } = props;
   const [open, setOpen] = React.useState(false);
-  const isInsideContent = React.useContext(ContentContext);
+  const { isInsideContent } = useContentContext(CONTEXT_MENU_NAME);
```

Comment:
I think whenever possible we do this no? `contentContext = ` and then later `contentContext.isInsideContent`

Reply:
Same in other places obviously if we decide to do it.

Reply:
Oh yeah, true, I tried to go with minimal changes and went a bit too minimal here 😆 will change for sure!